pkg/rebuild/maven/strategy.go

```diff
@@ -137,4 +145,20 @@ var toolkit = []*flow.Tool{
 				export PATH=$JAVA_HOME/bin:$PATH`[1:]),
 		}},
 	},
+	{
+		Name: "maven/setup-gradle",
+		Steps: []flow.Step{
+			{
+				Runs: textwrap.Dedent(`
+				if [ ! -f gradlew ]; then
+					wget -q -O gradle-8.14.3-bin.zip https://services.gradle.org/distributions/gradle-8.14.3-bin.zip
+					unzip -q gradle-8.14.3-bin.zip -d /opt/
+					rm gradle-8.14.3-bin.zip
+					export GRADLE_HOME=/opt/gradle-8.14.3
+					export PATH=$GRADLE_HOME/bin:$PATH
+				fi`[1:]),
+				Needs: []string{"zip", "wget"},
```

Comment:
both of these are in busybox so you can omit them for now.

Reply:
It fails to find `unzip` if I omit it. So we need `zip`.
Also, `wget` is not present in trixie-slim. It is there because it is a requirement of `setup-java`.